fuzz/cli.rs

```diff
@@ -161,12 +161,58 @@ fn run(fuzzer: Fuzzer, target: &str) -> Result<(), Error> {
 }
 
 /// Run one target fuzz test using AFL
-fn run_afl(_filter: &str) -> Result<(), Error> {
-    // AFL requires initial inputs, so leave it to future.
-    // General process:
+fn run_afl(target: &str) -> Result<(), Error> {
+    let fuzzer = Fuzzer::Afl;
+
     // 1. cargo afl build (in fuzzer-afl directory)
-    // 2. cargo afl fuzz -i in -o out target/debug/fuzzer-afl
-    unimplemented!()
+    let fuzzer_build = Command::new("cargo")
+        .args(&["afl", "build", "--bin", target, "--target-dir", "./target"])
```

Comment:
looks like `--target-dir` is not very necessary?

Reply:
without specifying target dir, this crate will share the `target` dir of the top level workspace, which is not expected IMHO.

Reply:
Yeah, in fact I was expecting that, but I don't have a strong opinion, as long as gitignore files are handled correctly.